tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -95,7 +97,7 @@ def index_impl(self, ctx, experiment=None):
                 }
         return result
 
-    def scalars_impl(self, ctx, tag, run, experiment, output_format):
+    def _scalars_impl(self, ctx, tag, run, experiment, output_format):
```

Comment:
No, this actually is public. The hparams plugin uses it:
<https://github.com/tensorflow/tensorboard/blob/9f87f60c317b8591dc4803b9d923616380d004b3/tensorboard/plugins/hparams/list_metric_evals.py#L54>

Whether it _should_ be public is a different question, but please don’t
change it in this patch.

Reply:
yep, crossed signals-- done already